client/src/js/screens/pages/Home/index.js

```diff
@@ -2,15 +2,34 @@ import React from 'react';
 import { Link } from 'react-router';
 import Footer from '../../components/Footer';
 import Navbar from '../../components/Navbar';
-import SignIn from '../../components/auth/SignIn';
 import RecipeCard from '../../components/RecipeCard';
-
+import RecipeCardLoader from '../../components/RecipeCardLoader';
 
 import logo from './../../../../assets/img/logo.png';
 import avatar from './../../../../assets/img/avatar.jpg';
 import bannerImage1 from '../../../../assets/img/banner-1.jpg';
 
 export default class Home extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      loading: true
+    }
+  }
+
+  async componentWillMount() {
+    try {
+      await this.props.getHomePageData();
+
+      this.setState({
+        loading: false
+      });
+    } catch (error) {
+      console.log(error);
```

Comment:
I don't think `console.log` is needed here especially when you are done with a file and pushed to Github. I'm not so sure but please look into this.